app/code/Magento/Store/App/Response/Redirect.php

```diff
@@ -177,12 +177,12 @@ public function updatePathParams(array $arguments)
     /**
      * Set redirect into response
      *
-     * @param \Magento\Framework\App\ResponseInterface $response
+     * @param \Magento\Framework\App\Response\HttpInterface $response
      * @param string $path
      * @param array $arguments
      * @return void
      */
-    public function redirect(\Magento\Framework\App\ResponseInterface $response, $path, $arguments = [])
+    public function redirect(\Magento\Framework\App\Response\HttpInterface $response, $path, $arguments = [])
```

Comment:
Unfortunately, we can't change the public method signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Reply:
Since public method signatures cannot be changed should I close this issue and pull request.

Reply:
@jasonevans1 unfortunately those changes cannot be accepted as they are not backward compatible